main/authorization.bs

```diff
@@ -19,3 +21,47 @@ the HTTP `Link` header with a `rel` value of `acl` [[#acl]].
 In the event that a server can't apply an ACL to a resource, it MUST deny
 access.
 [[Source](https://github.com/solid/specification/issues/130#issue-532777017)]
+
+Servers exposing client's access privileges on a resource URL MUST advertise
+by including the `WAC-Allow` HTTP header in the response of HTTP `HEAD` and
+`GET` requests.
+
+The syntax for the `WAC-Allow` header, using the ABNF syntax defined in
+Section 1.2 of [[!RFC7231]], is:
+
+```
+wac-allow           = "WAC-Allow" ":" OWS #access-param OWS
+access-param        = permission-group OWS "=" OWS quoted-access-modes
+permission-group    = 1*ALPHA
+quoted-access-modes = DQUOTE OWS *1(access-mode *(RWS access-mode)) OWS DQUOTE
```

Comment:
I don't have a strong opinion about requiring or making DQUOTE optional since some HTTP headers include parameters with double quotes for the value and some don't. Requiring it would be simpler indeed and I do have a preference for it.

*If* we change it to optional, the guidance for servers should be to use double quotes, and clients should be able parse with and without double quotes.

Reply:
Check, lets allow either.

Reply:
Done. If anyone disagrees or would like to provide implementation feedback regarding optional quotes, please raise an issue.